index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dreamers</title>
+</head>
+<body>
+    <h1> Dreamers </h1>
+</body>
+</html>
```

Comment:
@skoulas please add at the end of the line a new line at the end of the file. 
@neotherapper please confirm.

Reply:
@horacekalex i confirm